python/GangaDirac/test/Unit/Utilities/TestDiracUtilitiesExternals.py

```diff
@@ -0,0 +1,78 @@
+import os
```

Comment:
Really I think that *all* these tests should be marked as external since they are using the `GangaDirac` version of `execute`. If there are any tests which are not testing DIRAC they should be moved to `Ganga` and test against the code version of `execute` in `Ganga/Utility/execute.py`. We then end up with:

1. `GangaDirac/test/Unit/Utilities/TestDiracUtilities.py` which are *all* marked as `@external` and only contains the tests/asserts which relate to DIRAC
2. `Ganga/test/Unit/Utility/TestExecute.py` which are *not* marked as `@external` and contain all the timeout/cwd/etc. tests.